forc-plugins/forc-crypto/src/address.rs

```diff
@@ -4,11 +4,16 @@ use fuels_core::types::bech32::Bech32Address;
 use serde_json::json;
 use std::str::{from_utf8, FromStr};
 
+forc::cli_examples! {
+    [ Convert an address to another format => crypto "address fuel12e0xwx34nfp7jrzvn9mp5qkac3yvp7h8fx37ghl7klf82vv2wkys6wd523" ]
+}
+
 #[derive(Debug, clap::Args)]
 #[clap(
     author,
     version,
-    about = "Converts any valid address to all supported formats"
+    about = "Converts an address to another format",
+    after_long_help = help(),
```

Comment:
This doesn't seem to be working for me. 

```
❯ forc-crypto address -h                         
forc-crypto-address 0.48.1
Fuel Labs <contact@fuel.sh>  <--- Not sure why this is getting printed here?
Converts an address to another format

USAGE:
    forc-crypto address <ADDRESS>

ARGS:
    <ADDRESS>    The address to convert. It can be either a valid address in any format (Bech32
                 or hex)

OPTIONS:
    -h, --help       Print help information
    -V, --version    Print version information
```